lib/clouds/libcloud_common.py

```diff
@@ -370,30 +373,32 @@ def vmccleanup(self, obj_attr_list) :
                         if _running_instances :
                             sleep(int(obj_attr_list["update_frequency"]))
 
-                self.common_messages("VMC", obj_attr_list, "cleaning up vvs", 0, '')
+                if self.use_volumes :    
+                    self.common_messages("VMC", obj_attr_list, "cleaning up vvs", 0, '')
     
-                _running_volumes = True
-                while _running_volumes :
-                    _running_volumes = False
-                    for credentials_list in obj_attr_list["credentials"].split(","):
-                        credentials = credentials_list.split(":")
-                        tenant = credentials[0]
     
-                        _volumes = LibcloudCmds.catalogs.cbtool[credentials_list].list_volumes()
-                        for _volume in _volumes :
-                            if _volume.name.count("cb-" + obj_attr_list["username"]) :
-                                try :
-                                    cbdebug("Destroying: " + _volume.name + " (" + tenant + ")", True)
-                                    _volume.destroy()
-                                except :
-                                    pass
-                                _running_volumes = True
-                            else :
-                                _msg = "Cleaning up DigitalOcean. Ignoring volume: " + _volume.name
-                                cbdebug(_msg)
-    
-                        if _running_volumes :
-                            sleep(int(obj_attr_list["update_frequency"]))
+                    _running_volumes = True
+                    while _running_volumes :
+                        _running_volumes = False
+                        for credentials_list in obj_attr_list["credentials"].split(","):
+                            credentials = credentials_list.split(":")
+                            tenant = credentials[0]
+        
+                            _volumes = LibcloudCmds.catalogs.cbtool[credentials_list].list_volumes()
+                            for _volume in _volumes :
+                                if _volume.name.count("cb-" + obj_attr_list["username"]) :
+                                    try :
+                                        cbdebug("Destroying: " + _volume.name + " (" + tenant + ")", True)
+                                        _volume.destroy()
+                                    except :
+                                        pass
+                                    _running_volumes = True
+                                else :
+                                    _msg = "Cleaning up DigitalOcean. Ignoring volume: " + _volume.name
```

Comment:
Hmmm, that should not say DigitalOcean. Forgot to remove that when I wrote this. I'll add it to my next PR.